rocAL/include/api/rocal_api_meta_data.h

```diff
@@ -316,4 +316,9 @@ extern "C" void ROCAL_API_CALL rocalBoxIouMatcher(RocalContext p_context, std::v
  */
 extern "C" RocalTensorList ROCAL_API_CALL rocalGetMatchedIndices(RocalContext p_context);
 
+//TODO: Add the comments later
+extern "C" RocalMetaData ROCAL_API_CALL rocalCreateWebDatasetReader(RocalContext p_context, const char* source_path, const char* index_path, std::vector<std::set<std::string>> extensions, RocalMissingComponentsBehaviour missing_components_behavior, bool is_output);
+
+RocalMetaData ROCAL_API_CALL rocalGetAsciiDatas(RocalContext p_context);
```

Comment:
What's the use of this API? Why is it part of header exposed to user?

Reply:
This is similar to rocalGetImageLabels